app/models/post.rb

```diff
@@ -1,2 +1,19 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  after_save :update_posts_counter
+
+  default_scope -> { order(created_at: :desc) }
+
+  def five_recent_comments
+    comments.limit(5)
+  end
+
+  private
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly use the ActiveRecord `increment!` to update the post counter instead of increasing it manually with the update as it's cleaner and faster. For more info about `increment!` read this [docs](https://apidock.com/rails/ActiveRecord/Base/increment!)

Reply:
Thank you for the suggestion, @gemmen29. It does look cleaner to use `increment!` and is indeed faster. I'll definitely implement the change. 🙃